_docs_integrate/data-model-overview.md

```diff
@@ -610,6 +650,26 @@ In the example of Attribute succession, the peer can not decide to accept or rej
 
 NotificationItems are sent inside a Notification and specify which processes should be triggered when receiving the Notification.
 
+#### OwnSharedAttributeDeletedByOwnerNotificationItem
+
+An OwnSharedAttributeDeletedByOwnerNotificationItem will be sent, if an Attribute, a peer shared with you, was deleted by them.
+Internally, for your corresponding peer shared Attribute the `deletionInfo.deletionStatus` will be set to `"DeletedByOwner"`, if it wasn't set to `"ToBeDeleted` before.
+
+| Name        | Type                                                 | Description                                                         |
+| ----------- | ---------------------------------------------------- | ------------------------------------------------------------------- |
+| @type       | `"OwnSharedAttributeDeletedByOwnerNotificationItem"` |                                                                     |
+| attributeId | `string`                                             | The `id` of the shared LocalAttribute that was deleted by the owner |
+
+#### PeerSharedAttributeDeletedByPeerNotificationItem
+
+A PeerSharedAttributeDeletedByPeerNotificationItem will be sent, if an Attribute, you shared with a peer, was deleted by them.
```

Comment:
I'd explicitly write that you are the owner and that the peer has deleted the peer shared Attribute, for example: "If you are the owner of an Attribute and you have been shared it with one of your peers, a PeerSharedAttributeDeletedByPeerNotificationItem is sent to you when the peer shared Attribute has been deleted by them".